docs/book/src/developer/architecture/controllers/control-plane.md

```diff
@@ -229,19 +229,23 @@ The `status` object **may** define several fields:
 
 ## Example usage
 
-``` yaml
-kind: KubeadmControlPlane
-apiVersion: cluster.x-k8s.io/v1alpha3
-metadata:
-  name: kcp-1
-  namespace: default
-spec:
-  infrastructureTemplate:
-    name: kcp-infra-template
-    namespace: default
-  kubeadmConfigSpec:
-    clusterConfiguration:
-  version: v1.16.2
+```yaml
+apiVersion: v1
+items:
```

Comment:
Why using the list here compared to previous example?

Reply:
Good catch - I'll fix this up.